data/json/recipes/practice/melee.json

```
@@ -149,39 +149,492 @@
     "//1": "A bunch of duct tape and scrap to repair the dummy with each training session.",
     "components": [ [ [ "duct_tape", 50 ] ], [ [ "scrap", 20 ] ] ]
   },
+  {
+    "id": "prac_dancing",
+    "type": "practice",
+    "activity_level": "BRISK_EXERCISE",
+    "category": "CC_PRACTICE",
+    "subcategory": "CSC_PRACTICE_COMBAT",
+    "name": "dancing",
+    "description": "Practice some dancing moves with your imaginary partner.",
+    "skill_used": "dodge",
+    "time": "1 h",
+    "practice_data": { "min_difficulty": 0, "max_difficulty": 1, "skill_limit": 2 },
+    "book_learn": [ [ "mag_dodge", 0 ], [ "manual_dodge", 0 ] ]
```

```suggestion
    "book_learn": [ [ "mag_dodge", 0 ], [ "manual_dodge", 0 ] ],
    "autolearn"; [ [ "dodge", 2 ]]
```
Dancing wasn't invented post-literacy :p